### ISSUE TITLE
Add headersections to addon list/card page + table of contents

### ISSUE THREAD

**Issue by penguinmoose** [ScratchAddons/website-v2#253]
**Problem**
There are a lot of addons. This makes the addons page very long and hard to find a specific addon that would help the user - there are 140+ addons displayed on the page without any organization.

**Proposed solution**
There should be some kind of organization, such as headers for each type of addon - reccomended, editor, website (_projects, studios, profiles, etc_) - you name them. Someone quickly scrolling through the addons would be able to see the heading and know what kind of addon the following ones are. Additionally, there should also be a table of contents at the top, similar to what shows when “docs navigation” in the docs is pressed on mobile.

**Alternatives** - _1 and 2 can be used together_
- The table of contents can be in a box and text can wrap around it, like Wikipedia ones.
- The headings can be opened and closed - a dropdown section.
- The headings can be omitted entirely, as long as there is a break or a line separating sections.

**Comment by Hans5958** [comment]
Good idea. I would try to subdivide it based on the extension's categories, then I would try to find a good design for the TOC.
